nav2_route/src/route_server.cpp

```diff
@@ -100,6 +101,7 @@ nav2_util::CallbackReturn
 RouteServer::on_activate(const rclcpp_lifecycle::State & /*state*/)
 {
   RCLCPP_INFO(get_logger(), "Activating");
+  goal_intent_extractor_->activate();
```

Comment:
Deactivate, reset on cleanup, etc?

Reply:
Sorry, I was unclear. My comment is more: why? If we have an `activate`, shouldn't we at least have the complimentary `deactivate` to undo whatever `activate` did? If this is more just an `initialize` that doesn't need deactivation, rename to `initialize` so we don't misinterpret this as a lifecycle necessary thing.